packages/wrangler/src/__tests__/deploy.test.ts

```diff
@@ -12034,30 +12034,20 @@ export default{
 				main: "index.py",
 				compatibility_flags: ["python_workers"],
 			});
-			await fs.promises.writeFile(
-				"index.py",
-				"from js import Response;\ndef fetch(request):\n return Response.new('hello')"
-			);
+			const expectedModules = {
+				"index.py":
+					"from js import Response;\ndef fetch(request):\n return Response.new('hello')",
+			};
+			await fs.promises.writeFile("index.py", expectedModules["index.py"]);
 			mockSubDomainRequest();
 			mockUploadWorkerRequest({
 				expectedMainModule: "index.py",
+				expectedModules,
```

Comment:
This line proves that the actual module is included in the bundle, and that we don't rely upon the temp file in the `.wrangler/tmp` directory.